test/runtests.jl

```diff
@@ -212,16 +212,19 @@ mkpath(TEST_EXAMPLES_DIR)
     @testset "get scalar cell data" begin
       cell_data_raw = get_data(get_cell_data(vtk)[cell_data_name])
       cell_data_reshaped = reshape(cell_data_raw, ((Nx-1), (Ny-1), (Nz-1)))
-      @test cell_data_reshaped == cell_scalar_field
+      cell_data_reshaped1 = get_data_reshaped(get_cell_data(vtk)[cell_data_name], cell_data=true)
+      
+      @test cell_data_reshaped == cell_scalar_field 
```

Comment:
```suggestion
      @test cell_data_reshaped == cell_scalar_field
      @test cell_data_reshaped1 == cell_scalar_field
```
I think you forgot to add a test for your newly retrieved data, or am I missing something?